docker/k8s/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine:latest
+
+MAINTAINER sre@shedul.com
+
+RUN apk add --update python python-dev py-pip build-base postgresql-dev git && pip install pyyaml psycopg2 && rm -rf /var/cache/apk/*
+RUN git clone https://github.com/surgeventures/newrelic-plugin-agent.git src
```

Comment:
Can you just add the sources from the working directory? Dockerfile is in the same repository.

Reply:
ok